src/storm/utility/Extremum.cpp

```diff
@@ -133,7 +133,14 @@ void Extremum<Dir, ValueType>::reset() {
 
 template class Extremum<storm::OptimizationDirection::Minimize, double>;
 template class Extremum<storm::OptimizationDirection::Maximize, double>;
-template class Extremum<storm::OptimizationDirection::Minimize, storm::RationalNumber>;
-template class Extremum<storm::OptimizationDirection::Maximize, storm::RationalNumber>;
+
+#if defined(STORM_HAVE_CLN)
+template class Extremum<storm::OptimizationDirection::Minimize, storm::ClnRationalNumber>;
```

Comment:
This feels very strange here. Don't you want to have coefficients of the rational function rather than hardcoded types of rationals?